src/client/GameSave.cpp

```diff
@@ -2551,6 +2557,18 @@ char * GameSave::serialiseOPS(unsigned int & dataLength)
 	bson_append_string(&b, "platform", IDENT_PLATFORM);
 	bson_append_string(&b, "builtType", IDENT_BUILD);
 	bson_append_finish_object(&b);
+	if (fabsf(ambientAirTemp - (R_TEMP + 273.15f)) > 0.0001f)
```

Comment:
I don't think it was doing anything. The minimum version would already have been stored and I just moved it along with the new test. It doesn't affect anything else here otherwise.